static/app/components/sidebar/broadcastSdkUpdates.tsx

```diff
@@ -47,50 +118,31 @@ const BroadcastSdkUpdates = ({projects, sdkUpdates}: Props) => {
         'We recommend updating the following SDKs to make sure you’re getting all the data you need.'
       )}
     >
+      {!!deprecatedRavenSdkUpdates.length && (
+        <StyledAlert type="warning" icon={<IconWarning />}>
+          {tct(
+            `[first-sentence]. Any SDK that has the package name ‘raven’ may be missing data. Migrate to the latest SDK version.`,
+            {
+              ['first-sentence']: tn(
+                'You have %s project using a deprecated version of the Sentry client',
```

Comment:
@priscilawebdev I would put the `.` as part of the translation so it translates the full sentence.